src/pages/STG_PanelSchedule.page

```diff
@@ -108,6 +108,18 @@
                     </div>
                 </div>
             </div>
+
+            <div class="slds-grid slds-gutters_small">
+                <div class="slds-col slds-has-flexi-truncate slds-grid" role="listitem">
+                    <div class="slds-form-element">
+                        <apex:outputLabel value="{!$ObjectType.npo02__Households_Settings__c.Fields.Seasonal_Addresses_Batch_Size__c.Label}" for="seasonalAddressBatchSize" styleClass="slds-form-element__label" />
+                        <div class="slds-form-element__control">
+                            <apex:outputField value="{!stgService.stgHH.Seasonal_Addresses_Batch_Size__c}" rendered="{!isReadOnlyMode}" styleClass="slds-form-element__static" />
```

Comment:
The new "Seasonal Address Batch Size" is set to be the last field in the list. Is this fine or the field should be reordered? @force2b , @jesslopez-sf 

<img width="896" alt="Screen Shot 2019-07-23 at 4 21 06 PM" src="https://user-images.githubusercontent.com/28743286/61753853-badf2880-ad65-11e9-9f3a-f2fb513aec64.png">

Reply:
I think the last field is fine. If I had my preference, this field and Level Assignment Batch would have its own heading rather than be listed with the Rollup Batch Sizes.

Reply:
Thanks, @RadSFDeveloper ! I agree completely with @force2b .